$ libtoolize
libtoolize: putting auxiliary files in `.'.
libtoolize: linking file `./ltmain.sh'
libtoolize: Consider adding `AC_CONFIG_MACRO_DIR([m4])' to configure.in and
libtoolize: rerunning libtoolize, to keep the correct libtool macros in-tree.
libtoolize: Consider adding `-I m4' to ACLOCAL_AMFLAGS in Makefile.am.

diff --git a/ltmain.sh b/ltmain.sh
new file mode 120000
index 0000000..4159958
--- /dev/null
+++ b/ltmain.sh
@@ -0,0 +1 @@
+/usr/share/libtool/config/ltmain.sh
\ No newline at end of file
